src/libraries/System.Runtime.Numerics/tests/BigInteger/op_rightshift.cs

```diff
@@ -25,6 +25,54 @@ public static void BigShiftsTest()
             }
         }
 
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.Is64BitProcess))]
```

Comment:
No need to reset the PR, but we'd normally add a little comment like 
```suggestion
        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.Is64BitProcess))] // Requires over 2GB
```
or whatever the reason is.

Reply:
```suggestion
        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.Is64BitProcess))] // May fail on 32-bit due to a large memory requirement
```

Sounds good, added a comment